src/requestforattestation/RequestForAttestation.ts

```diff
@@ -164,6 +228,15 @@ export default class RequestForAttestation implements IRequestForAttestation {
     return this.verifySignature()
   }
 
+  /**
+   * Verifies the signature inside the [[RequestForAttestation]] object.
```

Comment:
Could we briefly explain here what it means "business-wise" if the signature is wrong?